### PR TITLE
[129]refactor: modify schemas

### PR DESCRIPTION
**Pull request by connorpobrien**
### Pull Request Summary

Issue 129: Modify schemas to match nonprofit specifications

- Modified attributes for recipientSchema, dropoffLogSchema, and pickuplogSchema
- Created new fieldSchema

### Pull Request Checklist

- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [x] The developer name is specified
- [x] The summary is completed
- [x] Ask for a review in communication channels

### Reviewer Checklist - PULL REQUEST REVIEWER ONLY

IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized pull request reviewers. If you are the individual contributor, do not fill out the rest of the fields or check the boxes.

## Reviewer: Emma Sauerborn and Ishaan Sathaye

NOTE: Pull requests should only be merged when all boxes are checked.

- [x] Assign yourself as reviewer if not assigned
- [x] The reviewer name is specified
- [ ] The code is fully reviewed
- [x] Meaningful feedback is given
- [x] Let developer know a review has been made

### CODE REVIEW

File: backend/models/fieldSchema.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose')
+
+const fieldSchema = new mongoose.Schema(
+  {
+    options: [String],
```
Comment (esauerbo) [inline backend/models/fieldSchema.js:6]: Sorry this wasn't included in the spec but can you add `name: String`

File: backend/models/dropoffLogSchema.js [hunk 2]
```diff
@@ -11,9 +11,8 @@ const DropoffLogSchema = new mongoose.Schema(
     name: String,
     recipientEntityType: String,
     demographic: String,
-    foodType: String,
     area: String,
-    lbsDroppedOff: Number,
+    foodAllocation: {String, Number},
```
Comment (esauerbo) [inline backend/models/dropoffLogSchema.js]: Can you change this to be a Map data type 
https://mongoosejs.com/docs/schematypes.html#maps

File: backend/models/pickupLogSchema.js [hunk 3]
```diff
@@ -10,9 +10,8 @@ const PickupLogSchema = new mongoose.Schema(
     vehicle: String,
     name: String,
     donorEntityType: String,
-    foodType: String,
     area: String,
-    lbsPickedUp: Number,
+    foodAllocation: {String, Number},
```
Comment (esauerbo) [inline backend/models/pickupLogSchema.js]: Change to map

**Review by esauerbo — COMMENTED** [review]
Just a few minor changes but great work on this!

**Comment by connorpobrien** [comment]
fixed ✅

**Review by esauerbo — APPROVED** [review]
great work!

**Review by ishaansathaye — APPROVED** [review]

**Pull request merged**